### ISSUE TITLE
Data Registration - Data Model - interop:updatedBy

### ISSUE THREAD

**Issue by joshdcollins** [solid/data-interoperability-panel#73]
Issue created following conversation in December 8th Interop Panel:

Suggestion to add a statement `interop:updatedBy` to pair with the existing `interop:updatedAt` statement when a DataRegistration is updated.

**Comment by justinwb** [comment]
Makes sense to add this - will include in update and leave this open in meantime.
